.gitlab/test.yml

```diff
@@ -0,0 +1,33 @@
+test:
+  extends: .check_skip_rust
+  timeout: 2 hours
+  image: paritytech/ci-unified:bullseye-1.77.0
```

Comment:
Let's keep 1.70.0 for all of them. We'll bump them up in a separate PR as we update our polkadot-sdk dependencies.